Codeinwp/neve
# feat: add aria-expanded for voice over #3864

### Summary

Registered a new JS function to handle the `aria-expanded` toggle when the mobile menu is being used.
I've done this to keep under the JS size limit, there is no more space for additional JS for the frontend. 
Since this is added only when the mobile menu is being used it should only register for those cases.

The function handles setting the `aria-expanded` attribute as specified here: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/examples/menu-button-links/

Added a new folder to store the current `accessibility` test and future tests that are linked to accessibility.

### Will affect visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/23024731/233054554-5333c9dc-f0e6-4355-884b-fdc094c3fafc.png)


### Test instructions

1. On a fresh instance or on a imported SS ensure the mobile menu is enabled. You can also add the **Menu Icon** component to open the side menu when not on mobile, but it is not required.
2. Check that without the menu being opened the menu button has the ARIA attribute `aria-expanded` set to `false`
    ![image](https://user-images.githubusercontent.com/23024731/233054762-a50dd711-23d0-49fa-a805-fedfe6959588.png)
3. When the menu is opened all menu buttons should contain the ARIA attribute `aria-expanded` set to `true`
4. Check that the attribute is set back to false when the menu is closed. E.g. by clicking the menu button or by clicking outside.
5. Check that the aria-attribute has the propper value even when the menu is closed from keyboard events. E.g. open with `Enter` and close with `Esc`.

## Check before Pull Request is ready:



Closes #3864.
